tsconfig.json

```diff
@@ -24,5 +24,10 @@
         "skipDefaultLibCheck": true, // Skip type checking .d.ts files that are included with TypeScript
         "downlevelIteration": true
     },
-    "exclude": ["node_modules", "build", "**/__tests__/**/*", "**/*.js", "**/*.jsx"] // *** The files to not type check ***
+    "exclude": ["node_modules", "build", "**/__tests__/**/*", "**/*.js", "**/*.jsx"],
+     "ts-node": {
+        "compilerOptions": {
+            "module": "commonjs"
+        }
+    }, // *** The files to not type check ***
```

Comment:
I think the comment should be next to exclude..
// *** The files to not type check ***